src/main/java/controller/MainController.java

```diff
@@ -95,27 +95,26 @@ public void validateFile(Popup_UploadFile uploadFile) {
 		if (returnValue == JFileChooser.APPROVE_OPTION) {
 			File selectedFile = jfc.getSelectedFile();
 			path = selectedFile.getAbsolutePath();
-
 			if (isValid(path)) {
```

Comment:
```suggestion
			
			if (isValid(path)) {
```

Reply:
keeping the white line there improves readibility

Reply:
Done :D